Python/bigquery/read_all_bigquery.py

```diff
@@ -19,16 +19,21 @@
 from apache_beam import Map
 from apache_beam.io import ReadAllFromBigQuery
 from apache_beam.io import ReadFromBigQueryRequest
+from apache_beam.options.pipeline_options import PipelineOptions
 
 
 def run(argv=None):
+  """Use ReadAllFromBigQuery to define table and query reads from
+  BigQuery at pipeline runtime.
+  """
```

Comment:
nit

Do/should we have a linter? I have an impression that the standard is having a line break after `""""` when the comment is multi-line

Reply:
Depends on the lint styles. Let me add the linter later to format all these py code.